whylog/tests/tests_config/__init__.py

```diff
@@ -0,0 +1,18 @@
+from os.path import join
```

Comment:
`import os.path` and then `os.path.join(...)`. This is the way everyone seems to write it (for some reason) and because of that importing a single function makes code harder to read.